gen-athena-docs/main.go

```diff
@@ -0,0 +1,118 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"slices"
+	"strings"
+
+	"text/template"
+
+	"github.com/gravitational/teleport/gen/go/eventschema"
+)
+
+// colNameList prints an example of columns from a given Access Monitoring event
+// view to include an example command. It only prints up to the first three
+// columns.
+func colNameList(cols []*eventschema.ColumnSchemaDetails) string {
+	var names []string
+	for i := 0; i < len(cols) && i < 3; i++ {
+		names = append(names, cols[i].NameSQL())
+	}
+	return strings.Join(names, ",")
```

Comment:
It doesn't matter at all in this case since `i` is so small, but for illustration's sake here's another way you could write this that doesn't require allocating a slice that will be thrown away.

```suggestion
	var sb strings.Builder
	for i := range min(3, len(cols)) {
	    if i != 0 {
	        sb.WriteString(",")
	    }
	    sb.WriteString(cols[i].NameSQL())
	}
	return sb.String()
```